pkg/aspectsupport/uber/env_test.go

```diff
@@ -24,5 +24,5 @@ func TestEnv(t *testing.T) {
 	log := env.Logger()
 	log.Infof("Test%s", "ing")
 	log.Warningf("Test%s", "ing")
-	log.Errorf("Test%s", "ing")
+	_ = log.Errorf("Test%s", "ing")
```

Comment:
I guess... but the difference between "_ =" and dropping the error on the floor is not clear to me.  Isn't the improvement something like:

```
if err := log.Errorf(..); err != nil {
   t.Errorf("Errorf(): unexpected error %v", err) 
}
```